<launch>
    <node name="cone_detection_point" pkg="rr_iarrc" type="cone_detection_point" output="screen">

        <param name="orange_low_H" type="int" value="5" />
        <param name="orange_high_H" type="int" value="25" />
        <param name="orange_low_S" type="int" value="140" />
        <param name="orange_low_V" type="int" value="120" />

        <param name="canny_cut_min_threshold" type="int" value="30" />
        <param name="percent_max_distance_transform" type="double" value="0.7" />

        <!--The larger the height the lower it will be on the screen-->
        <param name="blockSky_height" type="int" value="200" />
        <param name="blockWheels_height" type="int" value="200" />
        <param name="blockBumper_height" type="int" value="200" />

        <param name="subscription_node" type="string" value="/camera/image_color_rect"/>
    </node>

</launch>
